Die App stürzt bei einer Texteingabe im Suchfeld ab

Xiaomi 12
HyperOS 2.0, Android 14

App-Version 2.2.2 Beta

Wie es im Titel steht, stürzt jedes Mal bei einer Texteingabe ab. Ich vermute, die App versucht im Internet bzw. in einer Anbieter DB (zB Deutsche Bahn) etwas zu finden und fliegt dabei raus.

Das Gleiche passiert, wenn ich "Zuhause" o. "Arbeit" festzulegen versuche.

Same here - can confirm that the issue still exist.

Device: Fairphone 5
OS: Android 14 (iodéOS 5.6-20241015-FP5)
App-Version: 2.2.2 - de.grobox.liberario

<details>
<summary>sccop - Transportr crash</summary>
`FATAL EXCEPTION: AsyncTask #1
Process: de.grobox.liberario, PID: 5391
java.lang.RuntimeException: An error occurred while executing doInBackground()
	at android.os.AsyncTask$4.done(AsyncTask.java:415)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:381)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:250)
	at java.util.concurrent.FutureTask.run(FutureTask.java:269)
	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
	at java.lang.Thread.run(Thread.java:1012)
Caused by: java.lang.NoClassDefFoundError: Failed resolution of: Lorg/slf4j/LoggerFactory;
	at de.schildbach.pte.AbstractHafasProvider.<clinit>(AbstractHafasProvider.java:55)
	at de.grobox.transportr.networks.TransportNetworksKt$networks$2.invoke(TransportNetworks.kt:53)
	at de.grobox.transportr.networks.TransportNetworksKt$networks$2.invoke(TransportNetworks.kt:39)
	at de.grobox.transportr.networks.TransportNetwork.getNetworkProviderReference(TransportNetwork.kt:51)
	at de.grobox.transportr.networks.TransportNetwork.access$getNetworkProviderReference(TransportNetwork.kt:32)
	at de.grobox.transportr.networks.TransportNetwork$networkProviderRef$2.invoke(TransportNetwork.kt:50)
	at de.grobox.transportr.networks.TransportNetwork$networkProviderRef$2.invoke(TransportNetwork.kt:50)
	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
	at de.grobox.transportr.networks.TransportNetwork.getNetworkProviderRef(TransportNetwork.kt:50)
	at de.grobox.transportr.networks.TransportNetwork.access$getNetworkProviderRef(TransportNetwork.kt:32)
	at de.grobox.transportr.networks.TransportNetwork$networkProvider$2.invoke(TransportNetwork.kt:49)
	at de.grobox.transportr.networks.TransportNetwork$networkProvider$2.invoke(TransportNetwork.kt:49)
	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
	at de.grobox.transportr.networks.TransportNetwork.getNetworkProvider(TransportNetwork.kt:49)
	at de.grobox.transportr.locations.SuggestLocationsTask.doInBackground(SuggestLocationsTask.java:48)
	at de.grobox.transportr.locations.SuggestLocationsTask.doInBackground(SuggestLocationsTask.java:31)
	at android.os.AsyncTask$3.call(AsyncTask.java:394)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	... 4 more
Caused by: java.lang.ClassNotFoundException: org.slf4j.LoggerFactory
	... 22 more

`
</details>

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -8,8 +8,8 @@ apply from: 'witness.gradle'
 android {
 
 	defaultConfig {
-		versionCode 126
-		versionName "2.2.2"
+		versionCode 127
+		versionName "2.2.3"
 
 		applicationId "de.grobox.liberario"
 		minSdkVersion 21

--- a/app/src/main/res/xml/changelog_master.xml
+++ b/app/src/main/res/xml/changelog_master.xml
@@ -3,6 +3,14 @@
     tools:ignore="UnusedResources">
 
     <release
+        version="2.2.3"
+        versioncode="127">
+        <change>IMPORTANT! Remove all remaining Navitia-based networks (due to shutdown of API): all regions in France, Spain, Italy and nationwide Australia network</change>
+        <change>Various UI fixes and improvements</change>
+        <change>Fix app crash</change>
+    </release>
+
+    <release
         version="2.2.2"
         versioncode="126">
         <change>IMPORTANT! Remove all remaining Navitia-based networks (due to shutdown of API): all regions in France, Spain, Italy and nationwide Australia network</change>

--- a/fastlane/metadata/android/en-US/changelogs/127.txt
+++ b/fastlane/metadata/android/en-US/changelogs/127.txt
@@ -1,0 +1,3 @@
+* IMPORTANT! Remove all remaining Navitia-based networks (due to shutdown of API): all regions in France, Spain, Italy and nationwide Australia network
+* Various UI fixes and improvements
+* Fix app crash

